src/client/hypertable/CephBroker.cc

```diff
@@ -46,11 +46,21 @@ std::atomic<int> CephBroker::ms_next_fd{0};
 /* A thread-safe version of strerror */
 static std::string cpp_strerror(int err)
 {
-  char buf[128];
   if (err < 0)
     err = -err;
   std::ostringstream oss;
-  oss << strerror_r(err, buf, sizeof(buf));
+  char error_buf[128] = "unknown error";
+  char *p;
+#ifdef STRERROR_R_CHAR_P
```

Comment:
this macro is defined by our own CMakeLists.txt.

while this file is supposed to be build as part of [hypertable](https://github.com/hypertable/hypertable) as a libcephfs client

see https://github.com/hypertable/hypertable/blob/master/src/cc/FsBroker/ceph/CephBroker.cc

@gregsfortytwo how shall we maintain CephBroker forwards ? or can we just remove it from our repo?

Reply:
Oh we can definitely kill CephBroker if it's causing issues; it probably doesn't work any more anyway.